File: Java/Euler.java
public class Euler {
	public static void main(String[] args) {
		float x, y, m, i, dy_dx, delta_y;
		x = input("Enter value for x: ");
		y = input("Enter value for f(x): ");
		m = input("Approximate what? [f(x)] : ");
		i = input("Enter increment: ");

		while (x <= m) {
			dy_dx = F(x, y);
			delta_y = find_delta_y(dy_dx, i);
			System.out.printf("x: %.6f\ty: %.6f\tdy/dx: %.6f\tΔy: %.6f%n", x, y, dy_dx, delta_y);
			y += delta_y;
			x += i;
		}
	}

	static float input(String message) {
		float number;
		System.out.print(message);
		number = new java.util.Scanner(System.in).nextFloat();
		return number;
	}

	static float F(float x, float y) {
		return (2 * x) + (2 * y);
	}

	static float find_delta_y(float dy_dx, float i) {
		return dy_dx * i;
	}
}

File: Julia/euler.jl
function main()
	print("Enter value for x: ")
	x = parse(Float64, readline())
	print("Enter value for f(x): ")
	y = parse(Float64, readline())
	print("Approximate what? [f(x)] : ")
	m = parse(Float64, readline())
	print("Enter increment: ")
	i = parse(Float64, readline())

	delta_y = 0.0
	dy_dx = 0

	while x <= m
		dy_dx = F(x, y)
		delta_y = find_slope(dy_dx, i)
		@printf "x:%f\ty:%f\tdy/dx:%f\tΔy:%f\n" x y dy_dx delta_y
		y = y + delta_y
		x = x + i
	end
end

function F(x, y)
	return 2x + 2y
end

function find_slope(dy_dx, i)
	return dy_dx * i
end

main()